admin-dev/themes/default/js/bundle/module/module.js

```diff
@@ -360,7 +356,7 @@ var AdminModuleController = function () {
           addRemoveLinks: true,
           dictDefaultMessage: '',
           hiddenInputContainer: _this.dropZoneImportZoneSelector,
-          addedfile: function(file) {
```

Comment:
If you can re-add this deletion as comment just before declaration it would be great, I left these on purpose to easily know callback's arguments if we need them in the future, without going through code and/or documentation